### ISSUE TITLE
Fedora-Minimal-armhfp-rpi-21-5-sda.raw.xz

### ISSUE THREAD

**Issue by Nolaan** [spartacus06/rpi2-fedora-image-builder#16]
Hello,
Is there a change you have a link for Fedora-Minimal-armhfp-rpi-21-5-sda.raw.xz F21 is EOL by now. Or maybe upload it on this repo with the release feature of Github?
Thanks

**Comment by sjenning** [comment]
Yes, F21 is EOL.  Unfortunately, it is also the last version of Fedora that used ext4 in the root filesystem (see #11).  There are two solutions I can think of 1) get rpi to include XFS as a built-in kernel module or 2) convert the XFS rootfs to an ext4.  ext4 is better on an SD card than XFS so I'm inclined to think that option 2 is the way forward.

I'll try to find some time to do this.
